database/index.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+const data = require('../sampleData.js');
+
+const connection = mongoose.connect('mongodb://localhost/musicPlayer', { useMongoClient: true });
```

Comment:
Question: is the useMongoClient param necessary and if so why? I didn't use that on mine and was wondering if we should all include/exclude it for consistency?